lib/dialects/mssql.ts

```diff
@@ -68,16 +68,20 @@ export function rawColumnToColumn(rawColumn: RawColumn): Column {
   }
 }
 
-export function parseDefaultValue(value: string | null) {
-  if (isNil(value)) return null;
+function parseDefaultValue(value: string | null) {
+  if (value === null) {
+    return null;
+  }
 
   while (value.startsWith('(') && value.endsWith(')')) {
     value = value.slice(1, -1);
   }
 
-  value = value.replace(/^\'([\s\S]*)\'$/, '$1');
+  if (value.trim().toLowerCase() === 'null') {
+    return null;
+  }
 
-  return isNaN(value as any) ? String(value) : Number(value);
+  return stripQuotes(value);
```

Comment:
I'm not sure why this converted the value to a number when the contract for RawColumn is `string | null`. I assume it's fine to leave it as string.